### ISSUE TITLE
DM Not working

### ISSUE THREAD

**Issue by superpep** [orakaro/rainbowstream#321]
I've tried this in MacOS Monterey and Kali Linux 2021.4

Whenever I run, for example `mes @dtvd88 hi`
It returns: *Sorry I can't understand.*

Also, can't see any DM. If I run `inbox <any num>` I get the same output which is: `<My name>  <My @> (x message) thread_id:1`

**Comment by DimitarDevP** [comment]
Did this get resolved? Is this still maintained?

**Comment by superpep** [comment]
Didn't try again. Is this happening to you currently?

**Comment by DimitarDevP** [comment]
Yes, I just installed it and it is very buggy. I cant send DMs or list my inbox thread properly. 
As I can see, this was last updated an year ago and that was minor changes. This essentially is built upon the Twitter API and if the API changes (as it does over time), the code needs to be updated.
